html/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/StyleScope.kt

```diff
@@ -42,11 +42,14 @@ interface StyleScope {
      * })
      * ```
      */
-    fun property(propertyName: String, value: StylePropertyValue)
+    fun property(propertyName: String, value: StylePropertyValue, important: Boolean)
```

Comment:
The new methods in an interface should have a default implementaiton to remain source compatible:

The existing implementations (I mean potential implementations not in this project) implement the methods with 2 parameters.

I see two options for the default implementations of new 3arg methods:
-  just throw an error - "!important is not supported by this implementation"
- call the 2arg overloads dropping the `important` parameter

Usually, when an error is throw then the problem will noticed much earlier. So I'd choose the first option. WDYT?

Then we just make sure that our implementation of the interface overrides the new 3arg methods rather then uses the default (throwing an error).